content/guides/anomalies.md

```diff
@@ -67,3 +66,29 @@ will not have their predictions be affected by short anomalies.
+
+Also, anomaly detection requires historical data to make good predictions. If you have only been collecting a metric for a few hours or a few days, anomaly detection probably won't be very useful.
+
+### Why does an anomaly "disappear" when I zoom in?
+
+At different zoom levels, the same query can result in time series with very different characteristics. When looking at longer time periods, each point represents the aggregate of many more-granular points. Therefore, each of these aggregate points may hide noise observed in the more granular points. For example, charts that show one week often appear smoother (less noisy) than charts that show just 10 minutes.
+
+The width of the grey band that is drawn by our anomaly detection algorithm is, in part, based on the noisiness of the time series in the plot. The band must be wide enough that ordinary noise is mostly inside the band and doesn't appear as anomalous. Unfortunately, when the band is wide enough to include ordinary noise, it might also be wide enough to hide some anomalies, especially when viewing short time windows.
+
+Here's a concrete example to illustrate. The `app.requests` metric is noisy but has a constant average value of 8. On one day, there is a 10-minute anomalous period, starting a 9:00, during which the metric has an average value of 10. The chart below shows this series in a graph with a one-day time window.
+
+<img src="/static/images/anomalies/disappearing_day.png" style="width:500px; border:1px solid #777777"/>
+
+The grey band here makes sense; it is wide enough to capture the noise in the time series. Yet, it is narrow enough that the anomaly at 9:00 stands out clearly. This next chart shows a zoomed-in view of a half-hour time window that includes the 10-minute anomaly.
+
+<img src="/static/images/anomalies/disappearing_half_hour.png" style="width:500px; border:1px solid #777777"/>
+
+Again, the band seems to be reasonably sized, because the non-anomalous data from 8:50 - 9:00 and from 9:10 - 9:20 is inside the band. A band any narrower would start to highlight normal data as anomalous. Notice the band in this graph is ~8x wider than the one in the previous graph. The anomalous period from 9:00 - 9:10 looks a little different from the rest of the series, but it is not extreme enough to fall outside of the band.
+
+In general, if an anomaly disappears when you zoom in, this doesn't mean that it's not an anomaly. It simply means that the individual points in the zoomed in view are not anomalous in isolation. Rather, the fact that many slightly unusual point occur together is what's anomalous.
```

Comment:
"It means that while the individual points in the zoomed-in view are not anomalous in isolation, the fact that many slightly unusual point occur together is what's anomalous."